src/main/java/core/basesyntax/service/operations/ReturnOperation.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public class ReturnOperation implements OperationHandler {

```

Comment:
The class `ReturnOperation` should not begin with an empty line. Please remove the redundant empty line at the start of the class definition.